src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,143 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidInputException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String CORRECT_LOGIN = "login555";
+    private static final String CORRECT_PASSWORD = "password";
+    private static final int CORRECT_AGE = 20;
+    private static User user;
+    private static User userTwo;
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        user = new User();
+        userTwo = new User();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user.setAge(CORRECT_AGE);
+        user.setLogin(CORRECT_LOGIN);
+        user.setPassword(CORRECT_PASSWORD);
+    }
+
+    @Test
+    void register_UserInStorage_Ok() {
```

Comment:
What does it mean - UserInStorage - I can not understand this state. LoginAlreadyExists ?